Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/StoreOperation/ADLSGen2FSOperation.java

```diff
@@ -119,4 +136,19 @@ public ADLSGen2FSOperation(@NotNull HttpObservable http) {
         return http.executeReqAndCheckStatus(req, 200, flushReqParams)
                 .map(ignore -> true);
     }
+
+
+    public class UriHelper {
```

Comment:
Looks like it may be static class?
```suggestion
    public static class UriHelper {
```